release-coredns

```diff
@@ -55,11 +55,11 @@ function latestGo {
     local VERSION="$2"
 
     if [[ -z "$VERSION" ]]; then
-        local LATEST=$(curl -s 'https://golang.org/dl/?mode=json' | jq -r '.[0].version')
-        local GOLANG=https://dl.google.com/go/${LATEST}.linux-amd64.tar.gz
+        local LATEST=$(curl -s 'https://go.dev/dl/?mode=json' | jq -r '.[0].version')
```

Comment:
for my local script I simply added a `-L` to curl, even with this new url that might be prudent to do

Reply:
```suggestion
        local LATEST=$(curl -sL 'https://go.dev/dl/?mode=json' | jq -r '.[0].version')
```